doc/tutorials/gapi/face_beautification/face_beautification.markdown

```diff
@@ -7,38 +7,34 @@
 # Introduction {#gapi_fb_intro}
 
 In this tutorial you will learn:
-* How to infer different networks inside the pipeline with G-API;
-* How to process a videostream inside the pipeline with G-API;
+* How to infer different networks inside a pipeline with G-API;
+* How to run a G-API pipeline on a video stream;
 * The main ideas of the Face Beautification algorithm.
 
 You can find the source code in
 `samples/cpp/tutorial_code/face_beautification.cpp`
 of the OpenCV library source tree.
 
-It is a well-known fact that geterogeneous algorithms based on Computer Vision
-image processing with the backup of deep neural networks inference
-(Deep Learning) are able to solve a huge variety of problems which CV-only
-ones might experience difficulties with. Let's consider the certain
-algorithm as an example.
-
 # Face Beautification algorithm {#gapi_fb_algorithm}
 
-The algorithm of retouching a face has been chosen as the subject for this
-tutorial. Its main idea is to implement a real-time
-videostream processing that detects faces and applies some filters to make them
-look beautiful (more or less). The pipeline is the following:
+We will implement a face beautification algorithm using a combination of modern
+Deep Learning techniques and traditional Computer Vision ones.
+The idea in general is to implement a real-time video stream processing that
+detects faces and applies some filters to make them look beautiful (more or
+less). The pipeline is the following:
```

Comment:
`look beautiful` is subjective, I'd put it as "detects faces and applies some automatic retouching to them"